train.py

```diff
@@ -87,7 +123,11 @@ def train(hparams) -> None:
         epilog="Trainer args docs can be found at PyTorch Lightning.")
 
     args = parser.parse_args()
-    args, _ = parser.parse_known_args()
+    if args.load_config:
+        with open(args.load_config, 'rt') as f:
+            t_args = argparse.Namespace()
+            t_args.__dict__.update(json.load(f))
+            args = parser.parse_args(namespace=t_args)
```

Comment:
is this overwriting all previous arguments or just those present in the JSON file?

Reply:
Any arg present in the Json file will override its default that’s written in the python scripts. You don’t need to specify all args. But even if you load args from json, if you type the arg on the command line it will override json.